api/doc/doc_test.go

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) 2024 The VeChainThor developers
+
+// Distributed under the GNU Lesser General Public License v3.0 software license, see the accompanying
+// file LICENSE or <https://www.gnu.org/licenses/lgpl-3.0.html>
+
+package doc
+
+import (
+	"regexp"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestVersion(t *testing.T) {
+	validVersion := regexp.MustCompile(`^\d+(\.\d+){2}$`)
+
+	assert.True(t, validVersion.Match([]byte(Version())))
+}
```

Comment:
Can a comment be added for future reference ?

Reply:
Not sure what comment are you expecting, would you mind suggest the change?

Reply:
Just a comment explaining what this test does.
Having a second look now and I'm not 100% certain of what it's matching on. Seems to be a valid version string like "1.2.3" ?